cve_bin_tool/OutputEngine.py

```diff
@@ -67,43 +73,49 @@ def output_csv(self, outfile):
     def write_console(self, modulename, version, cve_number, cve_severity, outfile):
         """ Output Module, Version, CVE_Number, Severity to the console in tabular form"""
 
-        # Max space for each tabular cell
-        MODULENAME_MAX = 18
-        VERSION_MAX = 9
-        CVE_NUMBER_MAX = 18
-        CVE_SEVERITY_MAX = 9
-
         # Calculate length of -- modulename, version, cve_number, cve_severity
         MODULENAME_LEN = len(modulename)
```

Comment:
```suggestion
        modulename_len = len(modulename)
```

method local variables should be lowercase